aea/package_manager/v1.py

```diff
@@ -314,6 +296,11 @@ def json(
 
         return data
 
+    @staticmethod
+    def _load_packages(packages_file: Path) -> Dict[str, Dict[str, str]]:
+        """Load packages json file."""
+        return cast(Dict, json.loads(packages_file.read_text()))
```

Comment:
same thing

Reply:
@Karrenbelt same here:
if we want to move from json to yaml, it stays incapsulated here and easy to mock